tidymodel-prediction-workflows-inside-snowflake-with-orbital/tidymodel-prediction-workflows-inside-snowflake-with-orbital.md

```diff
@@ -0,0 +1,1145 @@
+
+<img src="assets/posit_workbench/01-start_new_session.png" style="width: 400px; height: auto;" />
+
+#### Step 2: Select an IDE
+
+When prompted, select RStudio Pro.
+
+<img src="assets/posit_workbench/02-create_new_session_snowflake.png" style="width: 400px; height: auto;" />
+
+#### Step 3: Log into your Snowflake account
+
+Next, connect to your Snowflake account from within Posit Workbench.
+Under `Session Credentials`, click the button with the Snowflake icon to sign in to Snowflake.
+Follow the sign in prompts.
+
+<img src="assets/posit_workbench/03-snowflake_login.png" style="width: 400px; height: auto;" />
+
+When you're successfully signed in to Snowflake, the Snowflake button will turn blue
+and there will be a check mark in the upper-left corner.
+
+<img src="assets/posit_workbench/04-snowflake_login_success.png" style="width: 400px; height: auto;" />
+
+#### Step 4: Launch the RStudio Pro IDE
+
+Click `Start Session` to launch the RStudio Pro IDE.
+
+Once everything is ready,
+you will be able to work with your Snowflake data
+in the familiar RStudio Pro IDE. Since the IDE is provided by the Posit Workbench Native App, 
+your entire analysis will run securely within Snowflake.
+
+![](assets/rstudio/01-rstudio.png)
+
+#### Step 5: Access the Quickstart Materials
+
+This Quickstart will walk you through the code contained in <https://github.com/posit-dev/snowflake-posit-quickstart-orbital/blob/main/fit_and_deploy.R>. To follow along, 
+open the file in your RStudio Pro IDE. There are two ways to do this:
+
+1. **Simple copy-and-paste** Go to File > New File > R Script and then copy the contents of [fit_and_deploy.R](https://github.com/posit-dev/snowflake-posit-quickstart-orbital/blob/main/fit_and_deploy.R) into your new file.
+2. **Starting a new project linked to the GitHub repo.** To do this:
+
+    1.  Go to `File` > `New Project` in the RStudio IDE menu bar.
+    
+    <img src="assets/rstudio/03-new-project.png" style="width: 400px; height: auto;" />
+
+    2.  Select Version Control in the New Project Wizard
+
+    <img src="assets/rstudio/04-project-wizard.png" style="width: 400px; height: auto;" />
+
+    3.  Select Git
+
+    <img src="assets/rstudio/05-git.png" style="width: 400px; height: auto;" />
+
+    4.  Paste the [URL](https://github.com/posit-dev/snowflake-posit-quickstart-orbital) of the GitHub repo and click Create Project
+
+    <img src="assets/rstudio/06-create-project.png" style="width: 400px; height: auto;" />
+
+    RStudio will clone a local copy of the materials on GitHub. You can use the Files pane in the bottom right-hand corner of the IDE to navigate to `quarto.qmd`. Click on the file to open it.
+
+> aside negative
+> 
+> SSH authentication is not available in Snowpark Container Services, so when creating projects from Git, you may need to authenticate Git operations over HTTPS, using a username and password or a personal access token.
```

Comment:
It's also possible to use [SSH over the HTTPS port](https://docs.github.com/en/authentication/troubleshooting-ssh/using-ssh-over-the-https-port), but that may be beyond the scope of this quickstart